ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -1593,16 +1596,30 @@
                 "public record SingleDataNodeParameter : DataNodeParameter {",
                 "\n    [DefaultValue(DefaultPremiumExperienceAdjustmentFactor)]",
                 "\n    [Range(0, 1, ErrorMessage = \"Value for {0} must be between {1} and {2}.\")]",
+                "\n    [Display(Order = 20)]",
                 "\n    public double PremiumAllocation { get; init; } = DefaultPremiumExperienceAdjustmentFactor;",
+                "\n",
+                "\n    [IdentityProperty]",
```

Comment:
Are these id properties? It should not be, imo, as they are the value of the parameter. Same for InterpolationMethod